src/context.ts

```diff
@@ -89,6 +72,7 @@ export class Context<E extends WebhookPayloadWithRepository = any>
    *
    */
   public repo<T>(object?: T) {
+    // @ts-expect-error
```

Comment:
I guess these are the points where we have to work on right? Do you know how to cast those generic webhooks @G-Rath @wolfy1339

Reply:
Yes, it would be nice to not require these

Reply:
This is because not all payloads have the `repository` property - your best bet would be to make a custom type guard that adjusts the type of payload (specifically in this case you could do it as an assertion guard).